site/public/js/polls.js

```diff
@@ -106,6 +106,106 @@ function updateDropdownStates(curr_state, cookie_key) {
     Cookies.set(cookie_key, !curr_state, { expires: expiration_date, path: '/' });
 }
 
+function togglePollFormOptions() {
+    const correct_options = $('.correct-box');
+
+    correct_options.each(function () {
+        $(this).prop('checked', $('#toggle-all').prop('checked'));
+    });
+}
+
+function validateCustomResponse() {
+    const custom_response = document.querySelector('.custom-poll-response');
+    const custom_response_submit = document.querySelector('.custom-response-submit');
+
+    const validate = () => {
+        if (custom_response.value.trim() !== '') {
+            custom_response_submit.disabled = false;
+        }
+        else {
+            custom_response_submit.disabled = true;
+        }
```

Comment:
This can be simplified:
```suggestion
        custom_response_submit.disabled = custom_response.value.trim() === '';
```